### PR TITLE
support MontanaInstruments Cryostation

### PR DESCRIPTION
**Pull request by julienbarrier**
* tested on Montana Instruments Cryostation s50
* using `IPInstrument` because this instrument cannot be found as a Visa resource

### CODE REVIEW

File: qcodes_contrib_drivers/drivers/MontanaInstruments/cryostation.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+from typing import Optional, Any, Dict
+
+from qcodes.instrument import IPInstrument
+from qcodes.parameters import Parameter
+from qcodes import validators as vals
+
+log = logging.getLogger(__name__)
+
+
+class MontanaInstruments_Cryostation(IPInstrument):
```
Comment (astafan8) [inline qcodes_contrib_drivers/drivers/MontanaInstruments/cryostation.py:21]: can't this instrument be connected to visa VISA? So the driver class inherits from VisaInstrument, and then for the adress you pass something like `TCPIP[board]::host address::port::SOCKET` (see here https://pyvisa.readthedocs.io/en/latest/introduction/names.html#visa-resource-syntax-and-examples )?

Comment (astafan8) [reply qcodes_contrib_drivers/drivers/MontanaInstruments/cryostation.py:21]: IPInstrument is not recommended to be used, unless really needed, it's an old "hack" implementation in a way, so if it's possible to use VISA that'd be much more robust in terms of connectivity as well.

**Review by astafan8 — APPROVED** [review]

File: qcodes_contrib_drivers/drivers/MontanaInstruments/cryostation.py (hunk 1 above)
Comment (julienbarrier) [reply qcodes_contrib_drivers/drivers/MontanaInstruments/cryostation.py:21]: Unfortunately this raises a VisaIOError. Trying to find the resource directly from pyvisa or NI Max is impossible.

Comment (astafan8) [reply qcodes_contrib_drivers/drivers/MontanaInstruments/cryostation.py:21]: ok :(

**Pull request merged**